packages/guilded.js/lib/structures/Message.ts

```diff
@@ -92,8 +92,8 @@ export class Message extends Base<ChatMessagePayload> {
         return this.serverId ? this.client.members.cache.get(buildMemberKey(this.serverId, this.authorId)) ?? null : null;
     }
 
-    /* Update message content */
-    update(newContent: RESTPostChannelMessagesBody | string): Promise<Message> {
+    /* Edit message content */
+    edit(newContent: RESTPostChannelMessagesBody | string): Promise<Message> {
```

Comment:
(need to import `Embed` too)
```suggestion
    edit(newContent: RESTPostChannelMessagesBody | Embed | string): Promise<Message> {
```